Userland/Libraries/LibJS/Runtime/RegExpConstructor.cpp

```diff
@@ -91,4 +198,84 @@ JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::symbol_species_getter)
     return vm.this_value();
 }
 
+// get RegExp.input https://github.com/tc39/proposal-regexp-legacy-features#get-regexpinput
+// get RegExp.$_ https://github.com/tc39/proposal-regexp-legacy-features#get-regexp_
```

Comment:
You'll need to implement all of these as separate functions so they don't end up pointing to the same accessor function objects.

i.e. `Object.getOwnPropertyDescriptor(RegExp, "input").get !== Object.getOwnPropertyDescriptor(RegExp, "$_").get` - please also add a test for that

Reply:
OK